src/main/java/ua/javarush/island/entity/Animal.java

```diff
@@ -0,0 +1,136 @@
+package ua.javarush.island.entity;
+
+import ua.javarush.island.gameisland.Area;
+import ua.javarush.island.gameisland.Coordinate;
+import ua.javarush.island.gameisland.Direction;
+import ua.javarush.island.gameisland.Island;
+import java.util.Map;
+import java.util.SplittableRandom;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.locks.Lock;
+
+public abstract class Animal extends Organism {
+
+    private boolean isMoved = false;
+    protected int speedOfMove;
```

Comment:
If this field is the same for all instances of one specific class should we use a field?